app/managers.py

```diff
@@ -0,0 +1,70 @@
+import psycopg2
+from psycopg2 import Error
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        try:
+            self._connection = psycopg2.connect(
+                host="localhost",
+                database="cinema",
+                user="postgres",
+                password="",
+                port=5432,
+            )
+
+            self._table_name = "actors"
+
+            print("Successful connect")
+        except (Exception, Error) as err:
+            print("Error", err)
+
+    def all(self) -> list[Actor]:
+        cursor = self._connection.cursor()
+
+        cursor.execute("SELECT * FROM actors")
+
+        return [
+            Actor(*row)
+            for row in cursor.fetchall()
+        ]
+
+    def create(self, first_name: str, last_name: str) -> None:
+        cursor = self._connection.cursor()
+
+        cursor.execute(
+            f"INSERT INTO "
+            f"{self._table_name} (first_name, last_name) "
+            f"VALUES (%s, %s)",
+            (first_name, last_name))
```

Comment:
please, use a code style as mentioned in this file:
https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md

The braces must be located in one of two options: open and start with the text or have a line break between the text